dora/core/common/pom.xml

```diff
@@ -173,6 +173,30 @@
       <version>${jupiter.version}</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
```

Comment:
we would like to put dependency in the root pom and submodules would rely on the root pom change